src/test_workflow/integ_test/service_opensearch_dashboards.py

```diff
@@ -0,0 +1,94 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import logging
+import os
+import subprocess
+import tarfile
+
+import requests
+import yaml
+
+from system.process import Process
+from test_workflow.integ_test.service import Service
+
+
+class ServiceOpenSearchDashboards(Service):
+    def __init__(
+        self,
+        manifest,
+        component_name,
```

Comment:
There's something that seems off with this interface. It should be representing the service, but yet it includes a component (name and test config). I'd expect the service to be just wrapping OpenSearch or OpenSearch Dashboards and know nothing about the component being tested.